lib/cardano_explorer/genesis.rb

```diff
@@ -0,0 +1,32 @@
+module CardanoExplorer
+  class Genesis < Base
+
+    def self.find
+      raise NoMethodError
+    end
+
+    def initialize
+    end
+
+    # NOTE: defined in the API docs, but returns 404
+    # https://cardanodocs.com/technical/explorer/api/#path--api-genesis-address-pages
+    #
+    # def pages(options = {})
+    #   valid_params = %w[page pageSize redeemeed]
+    #   options = options.keep_if { |k| valid_params.include?(k) }
+    #   get('address/pages', options).parsed_response['Right']
+    # end
```

Comment:
So it turns out that their docs are out of date. The route to get the genesis addresses by a page number is now actually this:
`https://cardanoexplorer.com/api/genesis/address?page=1450`

There's no need to implement this now, in fact you could remove this comment, and I'll be happy to merge the rest (commented out code becomes messy later on). We can deal with querying genesis addresses later, unless you're very ambitious.

Reply:
I see. Yeah okay, I've just removed the method for now. We can implement that later